webui/react/package.json

```diff
@@ -30,6 +30,7 @@
     "js-sha512": "^0.8.0",
     "plotly.js-basic-dist": "^1.54.6",
     "portable-fetch": "^3.0.0",
+    "pretty-bytes": "^5.3.0",
```

Comment:
missing the package-lock changes?

Reply:
I looks like `pretty-bytes` is already in the lock file.